web/app/Http/Middleware/TrimStrings.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Wikijump\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\TrimStrings as Middleware;
+
+class TrimStrings extends Middleware
+{
+    /**
+     * The names of the attributes that should not be trimmed.
+     *
+     * @var array
+     */
+    protected $except = [
+        'password',
+        'password_confirmation',
```

Comment:
Why would passwords (or password confirmation codes) have leading/trailing spaces?

Reply:
Not my code, but presumably because it's valid ASCII and a good way to increase the entropy of your password.